docs/pages/divider.js

```diff
@@ -1,64 +1,309 @@
 // @flow strict
 import type { Node } from 'react';
-import PropTable from '../components/PropTable.js';
 import PageHeader from '../components/PageHeader.js';
-import CardPage from '../components/CardPage.js';
 import MainSection from '../components/MainSection.js';
+import Page from '../components/Page.js';
 
-const cards: Array<Node> = [];
-const card = (c) => cards.push(c);
+export default function DividerPage(): Node {
+  return (
+    <Page title="Divider">
+      <PageHeader
+        name="Divider"
+        description="If you have two things that need to be separated, put a `Divider` between them."
+        defaultCode={`
+function Example() {
+  const Block = ({ title, text }) => (
+    <Flex direction="column" gap={2}>
+      <Heading size="sm">{title}</Heading>
+      <Text size="md">{text}</Text>
+    </Flex>
+  );
 
-card(
-  <PageHeader
-    name="Divider"
-    description="If you have two things that need to be separated, put a `Divider` between them."
-  />,
-);
+  return (
+    <Flex>
+      <Box paddingX={4}>
+        <Block title="Discover ideas" text="Use the searh bar to discover ideas, people and trends. Explore suggested topics or search for topics of your own." />
+      </Box>
+      <Divider />
+      <Box paddingX={4}>
+        <Block title="Create Pins" text="Upload an image from your computer or mobile device to create a Pin. You can also create Pins from images yu find online." />
+      </Box>
+    </Flex>
+  );
+}
+`}
+      />
+      <MainSection name="Usage guidelines">
```

Comment:
Can this use our standard "Do/Don't" format?

Reply:
DONE

Reply:
This still needs a "Don't", and part of this "Do" is referring to negative guidance. Can this be split into a "Do"/"Don't" pair?

Reply:
These are now just repeating the first Best Practices pair. Perhaps work with Kate to develop content that captures the original Usage Guidelines without repeating later content?

Reply:
DISCUSSED, CHANGED

Reply:
nit: if we're not pairing it with a "Don't", I think we can forgo the "Do". When not in counterpoint to anything, the "Do" is superfluous